jigasi/Dockerfile

```diff
@@ -11,8 +11,8 @@ LABEL org.opencontainers.image.documentation="https://jitsi.github.io/handbook/"
 ENV GOOGLE_APPLICATION_CREDENTIALS /config/key.json
 
 RUN apt-dpkg-wrap apt-get update && \
-    apt-dpkg-wrap apt-get install -y jigasi jq jitsi-autoscaler-sidecar && \
-    apt-cleanup
+apt-dpkg-wrap apt-get install -y openjdk-17-jre openjdk-17-jdk jigasi jq jitsi-autoscaler-sidecar && \
```

Comment:
Do we need the JDK for jvb or jibri? If not, can we drop it from base-java?

Reply:
not sure about JVB/jibri re: JDK but I do know without these we get the following exception for jigasi:

`Caused by: java.lang.ExceptionInInitializerError: Exception java.lang.UnsatisfiedLinkError: Can't load library: /usr/lib/jvm/java-17-openjdk-arm64/lib/libawt_xawt.so [in thread "ForkJoinPool.commonPool-worker-2"]`

Reply:
there is an error with headless one for jigasi, because of the renderers in the media impl. I will try these days debugging it.